### DISCUSSION TITLE
My school disabled inspect element, is there a way around this?

### DISCUSSION THREAD

**Discussion by RapidOnGfuel** [3kh0/ext-remover#434] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Please help as my school disabled ctrl+shift+u and inspect.... is there a way around this besides unenrolling your Chromebook?

**Comment by Aka-but-nice** [comment, marked as answer]
Well, there is a method to bypass control-shift-u for the site you want to view the code of, go to view-source: + the site you want to view

**Reply by Darcii** [reply]
how does that help?

**Reply by Aka-but-nice** [reply]
It doesn't, it was his fault he mentioned "Please help as my school disabled ctrl+shift+u" Now, I wouldn't have said anything otherwise. @RapidOnGfuel If your school allows bookmarklets, you could try firebug or eruda.

**Reply by Darcii** [reply]
> It doesn't, it was his fault he mentioned "Please help as my school disabled ctrl+shift+u" Now, I wouldn't have said anything otherwise. @RapidOnGfuel If your school allows bookmarklets, you could try firebug or eruda.

Can you send eruda, and i've tried other console bookmarklets however they dont seem to activate im assuming due to the websites type.

**Reply by Aka-but-nice** [reply]
Yes, it may be because of the websites type, also it may be because your school disabled bookmarklet's

**Reply by Aka-but-nice** [reply]
@Darcii Here's the code:   javascript:(function () { var script = document.createElement('script'); script.src="//cdn.jsdelivr.net/npm/eruda"; document.body.appendChild(script); script.onload = function () { eruda.init() } })();

**Reply by Darcii** [reply]
> Yes, it may be because of the websites type, also it may be because your school disabled bookmarklet's

My school didnt must be website type.

**Reply by Darcii** [reply]
> Yes, it may be because of the websites type, also it may be because your school disabled bookmarklet's

My school didnt must be website type.
